### ISSUE TITLE
how to load model trained by accelerate with fsdp.

### ISSUE THREAD

**Issue by shatealaboxiaowang** [pacman100/DHS-LLM-Workshop#25]
Hi dear:

I finetuned with accelerate with fsdp, but i  do not know how to load checkpoint to do inference, checkpoint output is as below:

checkpoin-100
  - optomizer_0
    - __0_0.distcp
    - __1_0.distcp
    - _-2_0,distcp
  - pytorch_model_fsdp_0
    - .metadata
    - __0_0.distcp
    - __1_0.distcp
    - _-2_0,distcp
  - rng_state_0.pth
  - rng_state_1.pth
  - rng_state_2.pth
  - scheduler.pt
  - trainer_state.json

Looking forward to your reply
